ci/experimental/build_docker.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -e
+#
+# Build Snowflake Python Connector in Docker
+# NOTES:
+#   - To compile only a specific version(s) pass in versions like: `./build_docker.sh "3.5 3.6"`
+set -o pipefail
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source $THIS_DIR/set_base_image.sh
+CONNECTOR_DIR="$(dirname "$( dirname "${THIS_DIR}")")"
+echo "conndir= $CONNECTOR_DIR, thisdir=$THIS_DIR"
```

Comment:
I don't think this adds any value, we use these constants everywhere, if they were wrong a lot of things would be broken.
I think we don't need to log them anymore